### DISCUSSION TITLE
where is changelog or whats new or recent changes file

### DISCUSSION THREAD

**Discussion by a-gyaanapan** [mozilla-mobile/fenix#21118] [Q&A]
i am unable to find changelog or whats new or recent changes file. where is it?

**Comment by ghost** [comment]
> i am unable to find changelog or whats new or recent changes file. where is it?

I need to see this too firefox team I m using nightly but unable to find new features.

**Reply by cadeyrn** [reply]
You can find the release notes here:
https://www.mozilla.org/en-US/firefox/android/92.0/releasenotes/

And if you really want to see *all* changes, including under-the-hood changes, you can go through the list of commits:
https://github.com/mozilla-mobile/fenix/commits/releases_v92.0.0

or https://github.com/mozilla-mobile/fenix/commits/main for Nightly.

**Reply by ghost** [reply]
> You can find the release notes here:
> https://www.mozilla.org/en-US/firefox/android/92.0/releasenotes/
> 
> And if you really want to see *all* changes, including under-the-hood changes, you can go through the list of commits:
> https://github.com/mozilla-mobile/fenix/commits/releases_v92.0.0
> 
> or https://github.com/mozilla-mobile/fenix/commits/main for Nightly.

Thanks man

**Comment by jonalmeida** [comment]
We would like to also have some system that keeps a list of changes that go into nightly. You can follow along in this issue when we make progress: https://github.com/mozilla-mobile/fenix/issues/19925
